spec/amp-html-responsive-attributes.md

```diff
@@ -0,0 +1,40 @@
+<!---
```

Comment:
This looks good. But to confirm: this doc file we'll end up on `amp.dev`, right? Or do we need to do something to make sure it does.

Reply:
If it is automatic, I would expect it to go to https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-responsive-attributes based on where [AMP's Layout system](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/?format=websites) and [AMP Action and events](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/?format=websites) are located (also in this `spec/` directory). @CrystalOnScript @sebastianbenz can one of you confirm?

Reply:
I've opened a PR that should add it to the imported documents for amp.dev here:
https://github.com/ampproject/amp.dev/pull/5422